frontend/test/pytest/test_template.py

```diff
@@ -471,7 +779,86 @@ def qft(basis_state):
     params = jnp.array([1, 0, 0])
     interpreted_fn = qml.QNode(qft, device)
     jitted_fn = qjit(interpreted_fn)
-    assert np.allclose(interpreted_fn(params), jitted_fn(params))
+
+    interpreted_result = interpreted_fn(params)
+    jitted_result = jitted_fn(params)
+
+    assert np.allclose(interpreted_result, jitted_result)
+
+
+def test_aqft(backend):
+    """Test AQFT."""
+
+    def aqft():
+        qml.X(0)
+        qml.Hadamard(1)
+        qml.AQFT(order=1, wires=range(3))
```

Comment:
Should we be able to qjit `aqft(order)`?
(Probably that implies update qml.AQFT)

Reply:
I think making `order` qjittable would require a substantial change to the AQFT decomposition. Since `order` is only used for control flow, and not computations it makes sense to mark it as static. I'll update the test so that `order` is passed as input, but with `static_argnums` set so it will be clear that it is not qjittable.